src/fittable_models.jl

```diff
@@ -51,4 +51,4 @@ function (z::FM90)(λ::Real)
     return exvebv
 end
 
-(z::FM90)(λ::Quantity) = z(ustrip(u"angstrom", λ)) * u"mag"
+(z::FM90)(λ::Quantity) = z(ustrip(u"angstrom", λ))
```

Comment:
Why? This makes the outputs no longer unitful.

Reply:
Fixed it!